pulp_python/tests/functional/api/test_crud_content_unit.py

```diff
@@ -139,11 +167,13 @@ def test_upload_requires_python(python_content_factory):
 @pytest.mark.parallel
 def test_upload_metadata_24_spec(python_content_factory):
```

Comment:
Can you add to the metadata_23_spec test to check for 2.3 fields like how you test for 2.4 fields here?

Reply:
2.3 did not introduce new fields, see https://packaging.python.org/en/latest/specifications/core-metadata/#history